javascript/modules/client/README.md

```diff
@@ -752,14 +758,15 @@ for await (const step of steps) {
   }
 }
 ```
+
 ### Checking if user can vote in an ERC20 proposal
 
 ```ts
 import {
+  CanVoteStep,
```

Comment:
Is there a can vote step? read operations should be able to complete without any internal pauses

Reply:
Yes, it was solved in the addressList but forgot about it in the erc20